source/find_getmore_killcursors_commands.rst

```diff
@@ -342,7 +342,12 @@ Exhaust
 
 This section only applies to drivers that support exhaust cursors.
 
-On server versions 3.6 and later drivers MUST use **OP_MSG** for exhaust cursors.
+On server versions 5.1 and later drivers SHOULD use **OP_MSG** for exhaust
+cursors. Server versions 5.1 and later do not support **OP_QUERY** for exhaust
+cursors.
```

Comment:
```suggestion
On server versions 5.1 and later drivers MUSTuse **OP_MSG** for exhaust
cursors. Server versions 5.1 and later do not support **OP_QUERY**.
```
I think "SHOULD" should be "MUST" given server versions 5.1 and newer do not support `OP_QUERY` _at all_, not just for exhaust cursors. Failure to comply would constitute a faulty driver implementation for server versions 5.1 and newer (exhaust cursor that _doesn't_ use OP_MSG is not a valid option).

Reply:
Good catch.

🤦 I thought to making this a `SHOULD` was required to permit the fallback behavior. But the fallback still uses OP_MSG.